Compilation warning on iOS

Hi, I'm getting a compilation warning on iOS after incorporating audioplayers 0.17.1 in my flutter app.

**Description**
I incorporated this plugin to play audio assets through AudioCache in my flutter app. I tested it in Android without any warning but when I moved to iOS I got a compilation warning which seem to be related with a deprecated method in class _MPMediaItemArtwork_. However, the app worked fine in both platforms.

**Code**
The following is the way I'm using audioplayers in my code:
```
import 'package:audioplayers/audio_cache.dart';

class _SherlockTecScaffoldState extends State<SherlockTecScaffold> with WidgetsBindingObserver {
   final ALARMA_APP_PRIMER_PLANO = "alarma_app_primer_plano.mp3";
   final reproductor = AudioCache();

   @override
   void initState() {
       super.initState();
       ...
       reproductor.load(ALARMA_APP_PRIMER_PLANO);

       Timer( Duration(seconds: 3), () {
          _traerNuevasAlarmas().then( (bandera) {
             if (bandera)
                reproductor.play(ALARMA_APP_PRIMER_PLANO);
          } );
       } );

    @override
    void dispose() {
        ...
        reproductor.clearCache();
        super.dispose();
    }

    Future<void> _generarAlertasDemo(Timer timer) async {
       ...
       reproductor.play(ALARMA_APP_PRIMER_PLANO);
    }
    ...
}
```

**Log Errors**
The output from Xcode when I compile on iOS is the following:
```
desarrollo@iMac-de-Mauro sherlock_tec % flutter run    
Downloading ios tools...                                           54.6s
Downloading ios-profile tools...                                   36.9s
Downloading ios-release tools...                                  234.8s
Launching lib/main.dart on iPhone 12 Pro in debug mode...
 
Running pod install...                                             12.6s
Running Xcode build...                                                  
 └─Compiling, linking and signing...                        13.4s
Xcode build done.                                           100.7s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **

Xcode's output:
↳
    /Users/desarrollo/.pub-cache/hosted/pub.dartlang.org/audioplayers-0.17.1/ios/Classes/NotificationsHandler.swift:174:75: warning:
    'init(image:)' was deprecated in iOS 10.0: Use -initWithBoundsSize:requestHandler:
                        let albumArt: MPMediaItemArtwork = MPMediaItemArtwork.init(image: artworkImage)
```

**Platforms**
- Android: worked fine.
  - OS version: Android 10
  - Device: Motorola g7 power (physical device)
  - flutter version: 1.22.4
  - audioplayers version: 0.17.1
  - App ran in debug mode.
- iOS: worked fine but I got a compilation warning.
  - OS version: iOS 14.2
  - Device: iPhone 12 Pro (simulator ran in macOS Catalina)
  - flutter version: 1.22.5
  - audioplayers version: 0.17.1
  - The app was compiled in debug and release modes.

--- a/packages/audioplayers/darwin/Classes/NotificationsHandler.swift
+++ b/packages/audioplayers/darwin/Classes/NotificationsHandler.swift
@@ -171,7 +171,9 @@ class NotificationsHandler {
             if let imageUrl = self?.imageUrl {
                 let artworkImage: UIImage? = NotificationsHandler.geneateImageFromUrl(urlString: imageUrl)
                 if let artworkImage = artworkImage {
-                    let albumArt: MPMediaItemArtwork = MPMediaItemArtwork.init(image: artworkImage)
+                    let albumArt: MPMediaItemArtwork = MPMediaItemArtwork.init(boundsSize: artworkImage.size, requestHandler: { (size) -> UIImage in
+                        return artworkImage
+                })
                     Logger.log("Will add custom album art")
                     playingInfo[MPMediaItemPropertyArtwork] = albumArt
                 }

--- a/packages/audioplayers/darwin/Classes/NotificationsHandler.swift
+++ b/packages/audioplayers/darwin/Classes/NotificationsHandler.swift
@@ -171,11 +171,16 @@ class NotificationsHandler {
             if let imageUrl = self?.imageUrl {
                 let artworkImage: UIImage? = NotificationsHandler.geneateImageFromUrl(urlString: imageUrl)
                 if let artworkImage = artworkImage {
+                    if #available(iOS 10, *) {
                     let albumArt: MPMediaItemArtwork = MPMediaItemArtwork.init(boundsSize: artworkImage.size, requestHandler: { (size) -> UIImage in
                         return artworkImage
                 })
+                        playingInfo[MPMediaItemPropertyArtwork] = albumArt
+                    } else {
+                        let albumArt: MPMediaItemArtwork = MPMediaItemArtwork.init(image: artworkImage)
+                        playingInfo[MPMediaItemPropertyArtwork] = albumArt
+                    }
                     Logger.log("Will add custom album art")
-                    playingInfo[MPMediaItemPropertyArtwork] = albumArt
                 }
             }
             

